TGM/src/main/java/com/minehut/tgm/modules/DeathModule.java

```diff
@@ -18,14 +16,16 @@
      * This is to prevent players from "combat logging"
      * by team swapping to avoid getting a death.
      */
-    @EventHandler
-    public void onTeamChange(TeamChangeEvent event) {
-        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
-            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
-                event.getPlayerContext().getPlayer().setHealth(0);
-            }
-        }
-    }
+
+    // not need because i prevent people changing teams during the game
+//    @EventHandler
+//    public void onTeamChange(TeamChangeEvent event) {
+//        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
+//            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
+//                event.getPlayerContext().getPlayer().setHealth(0);
+//            }
+//        }
+//    }
```

Comment:
okay now you know i made that change, you could have easily also noticed it while reviewing the code, so you can deny the pr if you want because of that change but i specifically made the change for infected (and it was easier to just disable it globally because it really isn't needed in general) because if your humans and a infected is killing you can just /join and respawn (killing would make you infected and people would complain because they "didn't know" so best to just disable it)